src/SConscript

```diff
@@ -61,6 +61,13 @@ Import('*')
 if env['CONF']['USE_EFENCE']:
     env.Append(LIBS=['efence'])
 
+# Check for capstone dependency. Currently, only arm ISA supports capstone
+# disassembler, we only allow building capstone-related object if arm is one
+# of the simulated ISAs.
+
+env['CONF']['USE_CAPSTONE'] = \
+    env['CONF']['HAVE_CAPSTONE'] and env['USE_ARM_ISA']
+
```

Comment:
Note: The options will need to be in the Kconfig file after #69 merged.

Reply:
The option `USE_CAPSTONE` is depends on the config option `HAVE_CAPSTONE` and `USE_ARM_ISA`. It should be set in the `src/cpu/Kconfig` rather than in the set in `SConscript` directly. 

https://github.com/gem5/gem5/blob/d94d6017b051d1b33d8950bddcbb89662fbec503/src/cpu/Kconfig#L26

For the config options dependency in the Kconfig, please see:

https://docs.kernel.org/kbuild/kconfig-language.html#menu-attributes